django/src/rdwatch_scoring/views/eval.py

```diff
@@ -0,0 +1,26 @@
+from ninja.pagination import RouterPaginated
+
+from django.db.models import CharField, Value
+from django.db.models.functions import Concat
+from django.http import HttpRequest
+
+from rdwatch_scoring.models import EvaluationRun
+
+router = RouterPaginated()
```

Comment:
Should this be updated for the case where there are more than 100 evaluations?  I did some stuff to default regions to 1000, should this be increased from the stock 100?

Reply:
I think that would be fine. 100 vs 1000 wouldn't make a huge difference i would think

Reply:
Oh, i see this is already done